### ISSUE TITLE
Expose external Cluster ID

### ISSUE THREAD

**Issue by mmazur** [openshift-online/ocm-cli#59]
The web ui operates on the, not sure what to call it, _External Cluster ID_, which can be searched for in the search box and is prominently displayed on a cluster's details page.

The cli tool neither outputs that ID, nor searches for it. This creates an additional step for having to use the web ui, when dealing with support issues.

(It'd be nice if terminology was somehow in sync with the web ui, but that's not a hard requirement, I guess.)

**Comment by jhernand** [comment]
@mmazur The CLI can search using that identifier. For example the generic _get_ command can be used like this:

```
$ ocm get /api/clusters_mgmt/v1/clusters --parameter search="external_id = '24e999bc-3103-4dd1-a18f-0962cc580a55'"
```

The more specific _cluster list_ command also accepts the `--parameter` option, so you can use it like this:

```
$ ocm cluster list --parameter search="external_id = '24e999bc-3103-4dd1-a18f-0962cc580a55'"
```

I acknowledge that is not very user friendly. I think that a possible way to make this more user friendly would be to change the `cluster describe` command so that it looks for cluster that have the given argument as name, identifier or external identifier. For example, it would be nice if this worked:

```
$ ocm cluster describe 24e999bc-3103-4dd1-a18f-0962cc580a55
```

Would you consider that enough to close this issue?

**Comment by jhernand** [comment]
This is addressed by #60. It will be included in the next release.
